editor/components/EnvironmentBanner.tsx

```diff
@@ -0,0 +1,32 @@
+import Alert from "react-bootstrap/Alert";
+import { FaCircleInfo, FaTriangleExclamation } from "react-icons/fa6";
+import AlignedLabel from "./AlignedLabel";
+
+/**
+ * Banner that displays environment information when not in production
+ */
+export default function EnvironmentBanner() {
+  const environment = process.env.NEXT_PUBLIC_APP_ENV;
+
+  if (environment === "production") return null;
+
+  const isLocal = environment === "local";
+  
+  return (
+    <Alert 
+      variant={isLocal ? "warning" : "info"} 
+      className="mb-0 rounded-0"
+    >
+      <AlignedLabel>
+        {isLocal ? (
+          <FaTriangleExclamation className="me-2" />
+        ) : (
+          <FaCircleInfo className="me-2" />
+        )}
+        <span>
+          This is a <b>{environment}</b> environment for testing purposes.
+        </span>
+      </AlignedLabel>
+    </Alert>
+  );
+} 
```

Comment:
If it's not too late to add a linefeed here at the end of this file for POSIX compliance -- but not at the expense of shipping.

Reply:
makes me wonder if this is an option in Prettier. I somehow have my code editor set up to add new lines at end of files so i could look into it.

Reply:
i didn't find my setting, but Prettier says [this about it](https://prettier.io/docs/rationale.html#empty-lines) so maybe my format on save is just doing that by default.

Reply:
yeah, my code editor is also set up to add new lines at the end of files so i dont ever have to think about it